internal/pkg/client/net/pull.go

```diff
@@ -111,8 +111,7 @@ func DownloadImage(ctx context.Context, filePath string, netURL string) error {
 	return nil
 }
 
-// pull will pull a http(s) image into the cache if directTo="", or a specific file if directTo is set.
-func pull(ctx context.Context, imgCache *cache.Handle, directTo, pullFrom string) (imagePath string, err error) {
+func head(pullFrom string) (hash string, err error) {
```

Comment:
How does it help to introduce this function if it is only used once?

Reply:
It was because I wanted to reuse the "pull" code, but in the end I was not able to anyway...

So all it accomplished was to make the functions more similar - but not identical (or shared)